app/git/utils.py

```diff
@@ -605,8 +740,7 @@ def patch_issue_comment(comment_id, owner, repo, comment):
     """Update a comment on an issue via patch."""
     gh_client = github_connect()
     try:
-        repo = gh_client.get_repo(f'{owner}/{repo}')
-        issue_comment = repo.get_comment(comment_id).edit(comment)
+        issue_comment = _get_issue_comment(gh_client, owner, repo, comment_id).edit(comment)
         return issue_comment
```

Comment:
@thelostone-mc @gdixon  Regarding the discussion in the standup yesterday: 

The code above doesn't work.
The reason: `repo.get_comment()` returns a commit comment: https://pygithub.readthedocs.io/en/latest/github_objects/Repository.html#github.Repository.Repository.get_comment

What we need is an issue comment, so we should do something like: `repo.get_issue().get_comment()`: https://pygithub.readthedocs.io/en/latest/github_objects/Issue.html#github.Issue.Issue.get_comment 

I can fix this. But my question is if we still need the features that this brings back. One of the features this will bring back are the reactions to comments on issue:

<img width="643" alt="image" src="https://user-images.githubusercontent.com/8137830/145968087-92f2f217-f1f5-49a1-988a-240488c59edb.png">

My question: do we still need this features? It seems to have been broken for a long time.